tests/vibe.db.redis.redis.pr2407/source/app.d

```diff
@@ -0,0 +1,40 @@
+import vibe.db.redis.redis;
+import vibe.core.core;
+import vibe.core.log;
+
+enum redisPort = 16379;
+
+void main()
+{
+	runTask(&redisServerMock);
+	runTask(&listenRedis);
+	runApplication();
+}
+
+void redisServerMock()
+{
+	listenTCP(redisPort, (conn) {
```

Comment:
I'd always recommend to use port 0 and use `.bindAddress.port` of the returned `TCPListener`, since it's possible that multiple CI jobs run in parallel on the same machine.

Reply:
I updated the test to use `.bindAddress.port`.
If you'd like me to squash the commits, just let me know.

Reply:
Looks good as-is!